src/components/Books.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
+
+const Books = () => {
+  const name = 'GEDEON';
+  const author = 'USHINDI';
+  return (
+    <section>
+      this will display a list of books!
+      <div>
+        <Book n={name} a={author} />
```

Comment:
- [ ] [OPTIONAL] As the previous CRR didn't say anything about it, I will put it as optional but, Please make sure that the book with name: 'GEDEON' author: 'USHINDI' is displayed on the web page. To fix it, n={name} should be name={name} and author={author} because in your code, you pass "n" and "a" as props.
![Capture web_4-6-2022_115052_localhost](https://user-images.githubusercontent.com/58552748/171992475-98f725ac-00a7-4b03-9a9b-1a37a21e7f94.jpeg)

Reply:
thank you very much!